python/ray/serve/scripts.py

```diff
@@ -729,6 +729,22 @@ def status(address: str, name: Optional[str]):
 def shutdown(address: str, yes: bool):
     warn_if_agent_address_set()
 
+    # check if the address is a valid Ray address
+    try:
+        # see what applications are deployed on the cluster
+        details = ServeSubmissionClient(address).get_serve_details()
```

Comment:
One of the problems the original gh issue is complaining about is that when the Ray cluster is running, but Serve isn't (i.e. there is no Serve controller), `serve shutdown` still says it is shutting down Serve. Can you add a check here for that, and change the log in that case? You can check `controller_info` in the fetched serve details to confirm whether Serve is running or not.

You can also test this by running locally:
```
ray start --head
serve start
serve shutdown
```
vs
```
ray start --head
serve shutdown
```